docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md

```diff
@@ -42,11 +42,11 @@ Riot.im) using [this link](https://matrix.to/#/#KusamaValidatorLounge:polkadot.b
 
 You can have a rough estimate on that by using the methods listed
 [here](../../general/faq.md#What-is-the-minimum-stake-necessary-to-be-elected-as-an-active-validator-?).
-Validators are elected based on [Phragmén's algorithm](../../learn/learn-phragmen.md). To be elected into the
-set, you need a minimum stake behind your validator. This stake can come from yourself or from
-[nominators](../../learn/learn-nominator.md). This means that as a minimum, you will need enough KSM to set up
-Stash and Controller [accounts](../../learn/learn-keys.md) with the existential deposit, plus a little extra for
-transaction fees. The rest can come from nominators.
+To be elected into the set, you need a minimum stake behind your validator. This stake can come from 
```

Comment:
We should add the minimum stake amounts here, in the computed-dict.json I only see min_nominator_intention in KSM. So we will need to add the Polkadot amount as well.

Reply:
Yeah, the minimum stake required to push a validator into the active set changes from era to era. What @alfarok is proposing here https://github.com/w3f/polkadot-wiki/issues/3533 might help render that value.

The minimum bond required by the validators on both Polkadot and Kusama is 0. That is why there are no variables for them in computed-dict.json I think.